ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/GetObjectsOnTape.java

```diff
@@ -46,23 +47,21 @@ public CliCommand init(final Arguments args) throws Exception {
     }
 
     @Override
-    public GetObjectsOnTapeResult call() throws IOException, CommandException {
+    public GetObjectsOnTapeResult call() {
         try {
 
             final GetBlobsOnTapeSpectraS3Response response
                         = getClient().getBlobsOnTapeSpectraS3(new GetBlobsOnTapeSpectraS3Request(null, this.tapeId));
 
             return new GetObjectsOnTapeResult(this.tapeId, response.getBulkObjectListResult().getObjects().iterator());
-        } catch (final FailedRequestException e) {
-            if(e.getStatusCode() == 500) {
-                throw new CommandException("Error: Cannot communicate with the remote DS3 appliance.", e);
-            }
-            else if(e.getStatusCode() == 404) {
-                throw new CommandException("Unknown tape: " + this.tapeId, e);
-            }
-            else {
-                throw new CommandException("Encountered an unknown error of ("+ e.getStatusCode() +") while accessing the remote DS3 appliance.", e);
+        } catch (final IOException e) {
+            if (CommandExceptionFactory.hasStatusCode(e, 404)) {
```

Comment:
This is what I said about this in the factory. You can provide a more tailored message:
GetObjectsOnTape failed (CommandException): Unknown tape '1234'

Instead of 
GetObjectsOnTape failed (CommandException): Entity not found.